### PR TITLE
Remove etcd_migrate and embedded2external

### PR DESCRIPTION
**Pull request by sdodson**
This commit removes all task files, plays, and
variables that are specific to etcd 2-3 migration and
embedded to external migration.

### CODE REVIEW

File: roles/openshift_control_plane/defaults/main.yml [hunk 1]
```diff
@@ -0,0 +1,151 @@
+---
```
Comment (michaelgugino) [inline roles/openshift_control_plane/defaults/main.yml]: Don't want this file.

Comment (sdodson) [reply roles/openshift_control_plane/defaults/main.yml]: nope, i think that's the only new file that slipped in.

**Comment by sdodson** [comment]
/hold
going to wait to pull this in after the first round of 3.9.z fixes

**Pull request merged**